### PR TITLE
add Generic Talks

### PR DESCRIPTION
**Pull request by cristaloleg**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -802,4 +802,17 @@ Mobile People Talks — подкаст про мобильную разрабо
 | :-------------------------- | -------------------------------------------- | ---------------------------------------------- |
 | Сергей Головин              | [@sgolovin ](https://www.t.me/sgolovin )     | [daynin](https://github.com/daynin)            |
 
-<br><hr><br>
\ No newline at end of file
+<br><hr><br>
+
+
+#### Generic Talks ([@GenericTalks](https://twitter.com/GenericTalks), [#GenericTalks](https://twitter.com/search?q=%23GenericTalks)): [Site](https://generictalks.com), [Telegram](https://t.me/generictalks), [SoundCloud](https://soundcloud.com/generictalks), [Github](https://github.com/generictalks), [iTunes](https://podcasts.apple.com/pl/podcast/generic-talks/id1489980299), [Spotify](https://open.spotify.com/show/3VZMfTLvm4pWOe5v2Pp36c).
```
Comment (php-coder) [inline README.md]: https://generictalks.com => 404

:-/

Comment (cristaloleg) [reply README.md]: currently in progress, stay tuned 😉

Comment (cristaloleg) [reply README.md]: done

**Comment by avevlad** [comment]
@cristaloleg спасибо!

**Pull request merged**